code/modules/uplink/uplink_items.dm

```diff
@@ -1235,6 +1235,22 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 8
 	exclude_modes = list(/datum/game_mode/nuclear) //you can't buy it in nuke, because the elite hardsuit costs the same while being better
 
+/datum/uplink_item/suits/hardsuit/cybersun
+	name = "Cybersun Hardsuit"
+	desc = "The long forgotten hardsuit made by Cybersun industries. \
```

Comment:
A, not the. Hardsuits like these are probably produced more than one, and there could also be more hardsuits made by Cybersun.
```suggestion
	desc = "A long forgotten hardsuit made by Cybersun industries. \
```